filters/color-overlay/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2013-2017 Mathew Groves, Chad Engler
```

Comment:
Please replace this with your name @mudcube, want to make sure you get credit for this since you created this new package.